centreon/tests/e2e/features/Dashboards/10-widget-metric-graph-configuration/index.ts

```diff
@@ -0,0 +1,395 @@
+  'the dashboard administrator user updates a threshold to a value beyond the default range of the Y-axis',
+  () => {
+    cy.get('input[type="radio"][value="custom"]').eq(1).click({ force: true });
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    })
+      .eq(1)
+      .clear();
+    cy.getByLabel({
+      label: 'Thresholds',
+      tag: 'input'
+    })
+      .eq(1)
+      .type('600', { force: true });
+  }
+);
+
+Then(
+  'the Y-axis of the Metrics Graph widget is updated to reflect the change in threshold',
+  () => {
+    cy.getByTestId({ testId: 'critical-line-600-tooltip' }).should('exist');
+  }
+);
+
+Given('a dashboard that includes a configured Metrics Graph widget', () => {
+  cy.insertDashboardWithMetricsGraphWidget(
+    dashboards.default,
+    metricsGraphWidget
+  );
+  cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+  cy.wait('@listAllDashboards');
+  cy.getByLabel({
+    label: 'view',
+    tag: 'button'
+  })
+    .contains(dashboards.default.name)
+    .click();
+});
+
+When(
+  'the dashboard administrator user duplicates the Metrics Graph widget',
+  () => {
+    cy.getByLabel({
+      label: 'Edit dashboard',
+      tag: 'button'
+    }).click();
+    cy.wait('@performanceData');
+    cy.getByTestId({ testId: 'MoreVertIcon' }).click();
+    cy.getByTestId({ testId: 'ContentCopyIcon' }).click();
+  }
+);
+
+Then('a second Metrics Graph widget is displayed on the dashboard', () => {
+  cy.getByTestId({ testId: 'Min' }).eq(1).should('exist');
+  cy.getByTestId({ testId: 'Max' }).eq(1).should('exist');
+  cy.getByTestId({ testId: 'Avg' }).eq(1).should('exist');
+});
+
+Then('the second widget has the same properties as the first widget', () => {
+  cy.getByTestId({ testId: 'critical-line-400-tooltip' }).eq(1).should('exist');
+  cy.getByTestId({ testId: 'warning-line-200-tooltip' }).eq(1).should('exist');
+});
+
+Given('a dashboard featuring two Metrics Graph widgets', () => {
+  cy.insertDashboardWithMetricsGraphWidget(
+    dashboards.default,
+    metricsGraphDoublecWidget
+  );
+  cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+  cy.wait('@listAllDashboards');
+  cy.getByLabel({
+    label: 'view',
+    tag: 'button'
+  })
+    .contains(dashboards.default.name)
+    .click();
+  cy.getByLabel({
+    label: 'Edit dashboard',
+    tag: 'button'
+  }).click();
+  cy.getByTestId({ testId: 'More actions' }).eq(0).click();
+  cy.wait('@performanceData');
+});
+
+When(
+  'the dashboard administrator user deletes one of the Metrics Graph widgets',
+  () => {
+    cy.getByTestId({ testId: 'DeleteIcon' }).click();
+    cy.getByLabel({
+      label: 'Delete',
+      tag: 'li'
+    }).realClick();
+  }
+);
+
+Then(
+  'only the contents of the other Metrics Graph widget are displayed',
+  () => {
+    cy.getByTestId({ testId: 'Min' }).should('exist');
+    cy.getByTestId({ testId: 'Max' }).should('exist');
+    cy.getByTestId({ testId: 'Avg' }).should('exist');
```

Comment:
you should probably check something along the lines of 
`cy.getByTestId({ testId: 'Min' }).eq(1).should('not.exist'); cy.getByTestId({ testId: 'Max' }).eq(1).should('not.exist'); cy.getByTestId({ testId: 'Avg' }).eq(1).should('not.exist');` 
as well